CatProducer/python/catTools_cff.py

```diff
@@ -92,6 +92,25 @@ def catTool(process, runOnMC=True, useMiniAOD=True):
         
         process.catJetsPuppi.src = cms.InputTag("patJetsPuppiUpdated")
         process.catJetsPuppi.setGenParticle = cms.bool(False)
+        
+        from PhysicsTools.SelectorUtils.tools.vid_id_tools import DataFormat,switchOnVIDPhotonIdProducer,setupAllVIDIdsInModule,setupVIDPhotonSelection            
+        switchOnVIDPhotonIdProducer(process, DataFormat.MiniAOD)
+
+        #Use IDs from https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_X/RecoEgamma/PhotonIdentification/python/Identification/ 
+        photon_ids = ['RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_25ns_V1_cff',
```

Comment:
In the MiniAODs the IDs that are stored are

PhotonCutBasedIDLoose 
PhotonCutBasedIDTight 
cutBasedPhotonID-PHYS14-PU20bx25-V2p1-standalone-loose 
cutBasedPhotonID-PHYS14-PU20bx25-V2p1-standalone-medium 
cutBasedPhotonID-PHYS14-PU20bx25-V2p1-standalone-tight 
cutBasedPhotonID-Spring15-50ns-V1-standalone-loose 
cutBasedPhotonID-Spring15-50ns-V1-standalone-medium 
cutBasedPhotonID-Spring15-50ns-V1-standalone-tight 
mvaPhoID-Spring15-25ns-nonTrig-V2p1-wp90 
mvaPhoID-Spring15-50ns-nonTrig-V2p1-wp90 

John

> On 8 Feb 2016, at 23:13, Jason Lee <notifications@github.com> wrote:
> 
> In CatProducer/python/catTools_cff.py <https://github.com/vallot/CATTools/pull/417#discussion_r52232183>:
> 
> > @@ -92,6 +92,25 @@ def catTool(process, runOnMC=True, useMiniAOD=True):
> >          
> >          process.catJetsPuppi.src = cms.InputTag("patJetsPuppiUpdated")
> >          process.catJetsPuppi.setGenParticle = cms.bool(False)
> > +        
> > +        from PhysicsTools.SelectorUtils.tools.vid_id_tools import DataFormat,switchOnVIDPhotonIdProducer,setupAllVIDIdsInModule,setupVIDPhotonSelection            
> > +        switchOnVIDPhotonIdProducer(process, DataFormat.MiniAOD)
> > +
> > +        #Use IDs from https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_X/RecoEgamma/PhotonIdentification/python/Identification/ 
> > +        photon_ids = ['RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_25ns_V1_cff',
> does this need to be rerun? 
> it should already be in the miniAODs
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/vallot/CATTools/pull/417/files#r52232183>.
>